eclair-core/src/main/scala/fr/acinq/eclair/router/RouteCalculation.scala

```diff
@@ -109,7 +109,7 @@ object RouteCalculation {
     // should be able to route the payment, so we'll compute an htlcMaximumMsat accordingly.
     // We could also get the channel capacity from the blockchain (since we have the shortChannelId) but that's more expensive.
     // We also need to make sure the channel isn't excluded by our heuristics.
-    val lastChannelCapacity = amount.max(RoutingHeuristics.CAPACITY_CHANNEL_LOW)
+    val lastChannelCapacity = (amount * 2).max(RoutingHeuristics.CAPACITY_CHANNEL_LOW)
```

Comment:
I'm not sure about this, except if we have a real reason behind this change.

Reply:
This indeed shouldn't be necessary, I reverted it in https://github.com/ACINQ/eclair/pull/1443/commits/aaf17b6572a9500e5edb2fed1c1917579fd9e98a